pkg/validate/validate.go

```diff
@@ -122,7 +122,12 @@ func Files(fileNames []string, validationRules []*validationrule.ValidationRule,
 				var excludedRules []string
 				excludedRulesText, ok := originalRule.Annotations[excludeAnnotationName]
 				if ok {
-					excludedRules = strings.Split(excludedRulesText, ",")
+					excludedRules = slices.DeleteFunc(strings.Split(excludedRulesText, ","), func(s string) bool {
```

Comment:
Deleting all spaces regardless of the placement feels a bit strong :thinking: 

Maybe I'd rather go with splitting, trimming space and dropping if empty string

that should solve any spaces around commas and possible empty strings such as `, , ` or trailing `foobar, ` without modifying the actual identifier

WDYT?